instana/instrumentation/sanic_inst.py

```diff
@@ -0,0 +1,138 @@
+            if status_code is not None:
+                if 500 <= int(status_code) <= 511:
+                    span.mark_as_errored()
+                span.set_tag('http.status_code', status_code)
+
+            if response.headers is not None:
+                async_tracer.inject(span.context, opentracing.Format.HTTP_HEADERS, response.headers)
+            response.headers['Server-Timing'] = "intid;desc=%s" % span.context.trace_id
+        except Exception:
+            logger.debug("send_wrapper: ", exc_info=True)
+
+
+    if hasattr(sanic.response.BaseHTTPResponse, "send"):
+        @wrapt.patch_function_wrapper('sanic.response', 'BaseHTTPResponse.send')
+        async def send_with_instana(wrapped, instance, args, kwargs):
+            span = async_tracer.active_span
+            if span is None:
+                await wrapped(*args, **kwargs)
+            else:
+                response_details(span=span, response=instance)
+                try:
+                    await wrapped(*args, **kwargs)
+                except Exception as exc:
+                    span.log_exception(exc)
+                    raise
+    else:
+        @wrapt.patch_function_wrapper('sanic.server', 'HttpProtocol.write_response')
+        def write_with_instana(wrapped, instance, args, kwargs):
+            response = args[0]
+            span = async_tracer.active_span
+            if span is None:
+                wrapped(*args, **kwargs)
+            else:
+                response_details(span=span, response=response)
+                try:
+                    wrapped(*args, **kwargs)
+                except Exception as exc:
+                    span.log_exception(exc)
+                    raise
+
+
+        @wrapt.patch_function_wrapper('sanic.server', 'HttpProtocol.stream_response')
+        async def stream_with_instana(wrapped, instance, args, kwargs):
+            response = args[0]
+            span = async_tracer.active_span
+            if span is None:
+                await wrapped(*args, **kwargs)
+            else:
+                response_details(span=span, response=response)
+                try:
+                    await wrapped(*args, **kwargs)
+                except Exception as exc:
+                    span.log_exception(exc)
+                    raise
+
+
+    @wrapt.patch_function_wrapper('sanic.app', 'Sanic.handle_request')
+    async def handle_request_with_instana(wrapped, instance, args, kwargs):
+
+        try:
+            request = args[0]
+            try:  # scheme attribute is calculated in the sanic handle_request method for v19, not yet present
+                if "http" not in request.scheme:
+                    return await wrapped(*args, **kwargs)
+            except AttributeError:
+                pass
+            headers = request.headers.copy()
+            ctx = async_tracer.extract(opentracing.Format.HTTP_HEADERS, headers)
+            with async_tracer.start_active_span("asgi", child_of=ctx) as tracing_scope:
```

Comment:
Can you change `tracing_scope` to `scope` here as well as in `asgi.py` - seems like we're using `scope` in most places for consistency